### PR TITLE
Accept financial contributions responsiveness

### PR DESCRIPTION
**Pull request by ericyoungson**
Accept financial contributions: responsiveness issue on medium screens

Resolve https://github.com/opencollective/opencollective/issues/3995

# Description

  Added minWidth property to box

Screen Shot 2021-04-23 at 7 58 55 AM

### CODE REVIEW

File: components/Location.js [hunk 1]
```diff
@@ -2,6 +2,10 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
 import styled from 'styled-components';
+<<<<<<< HEAD
```
Comment (SudharakaP) [inline components/Location.js:5]: @ericyoungson : Think these are introduced from a merge conflict. You'll need to resolve the conflicts before pushing. 🤔

File: components/accept-financial-contributions/ApplyToHost.js [hunk 2]
```diff
@@ -125,8 +125,13 @@ class ApplyToHost extends React.Component {
             px={[3, 5]}
             my={3}
           >
+<<<<<<< HEAD
```
Comment (SudharakaP) [inline components/accept-financial-contributions/ApplyToHost.js:128]: @ericyoungson : Same as above the merge from main branch probably didn't go well. You'll need to resolve these conflicts. 🤔

**Review by SudharakaP — CHANGES_REQUESTED** [review]

**Comment by SudharakaP** [comment]
@ericyoungson : FYI, I've also updated your description a bit to add the full link, https://github.com/opencollective/opencollective/issues/3995 instead of #3995. The #3995 only refers links in this repo; but if you want to link something on another repo you need to put the full link. 😄

File: workspace.code-workspace [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (SudharakaP) [inline workspace.code-workspace:1]: @ericyoungson : I don't see why this file was checked in. This is probably related to your IDE? 🤔

Comment (ericyoungson) [reply workspace.code-workspace:1]: I have no idea. I only add files individually to the repo--hold on...

Comment (ericyoungson) [reply workspace.code-workspace]: Ok, rookie mistake. I've added `*code-worskpace` to my *~/.gitignore_global* file. _Shouldn't_ happen again! ;-)

Comment (SudharakaP) [reply workspace.code-workspace]: No worries. 😄

**Comment by ericyoungson** [comment]
> @ericyoungson : FYI, I've also updated your description a bit to add the full link, https://github.com/opencollective/opencollective/issues/3995 instead of #3995. The #3995 only refers links in this repo; but if you want to link something on another repo you need to put the full link. 😄 

Noted; thanks!

**Review by Betree — APPROVED** [review]
The page still looks a bit strange on mid-size screens, but this definitely solves the issue reported in https://github.com/opencollective/opencollective/issues/3995. Thanks @ericyoungson!

**Comment by ericyoungson** [comment]
Cool. Maybe it needs some design review? @Betree Let me know if you have any other suggested changes, I'd be happy to follow up; especially now that I know where to look! ;-)

**Pull request merged**